backend/Makefile

```
@@ -42,9 +42,13 @@ install-dev: ## Install development dependencies (includes test)
 
 # Testing
 .PHONY: test
-test: install ## Run all tests (unit + integration)
-	@echo "$(GREEN)Running all tests...$(NC)"
-	$(PYTEST) $(TEST_DIR)/ -v --tb=short
+test: install ## Run all tests (unit + integration + e2e) with proper isolation
```

checkmake (0.2.2)

[warning] Target body for "test" exceeds allowed length of 5 (6).

(maxbodylength)